docs/configuration.rst

```diff
@@ -1663,6 +1663,26 @@ Example:
    The created ``needs.json`` file gets stored in the ``outdir`` of the current builder.
    So if ``html`` is used as builder, the final location is e.g. ``_build/html/needs.json``.
 
+.. _needs_build_needumls:
+
+needs_build_needumls
+~~~~~~~~~~~~~~~~~~~~
+
+Creates a folder during other builds, like ``html``.
+
+This option like :ref:`needs_build_json` allows to have one single Sphinx-Build for two output formats. 
+
+All the files that stores the generated plantuml code from :ref:`needuml` will be saved in this created folder
```

Comment:
You should mention what kind of data this option can take above.

First always describe how the user needs to configure something.
Then explain what it does (max. one sentence for the function as introduction as the beginning.)
I was confused first, has I thought it gets a boolean value.

Reply:
And keep the sentences more simple.
It's hard to follow the parts with "that"